### ISSUE TITLE
GLM method section

### ISSUE THREAD

**Issue by Remi-Gau** [nilearn/nilearn#4493]
Given I was working on methods reporting recently, I thought it would be nice if the GLM report (or maybe even the GLM first and second level class instances) contained a method section that could copy pasted into an article.

**Comment by bthirion** [comment]
Good idea. Do you have an example of what it would look like ?

**Comment by Remi-Gau** [comment]
This is the one I generate for an spm.based pipeline but even that one is work in progress.

I should go through the COBIDAS recommendations again to see if I am missing something.

https://bidspm.readthedocs.io/en/latest/output/method_section.html#output-example-glm-subject-level

**Comment by Remi-Gau** [comment]
Maybe worth synching with with the fitlins folks to see if they could be interested.

@effigies

**Comment by bthirion** [comment]
Yep, sounds good. I think it could integrate quite well to our reports.
